lib/Support/PathList.cpp

```diff
@@ -24,6 +27,32 @@ std::string getCurrentExecutableFullPath() {
   return FullPath.str().str();
 }
 
+std::string getCurrentRoot() {
```

Comment:
This function sometimes returns `""`, but callers do not handle such situation.
Let's abort in those case so we can guarantee that if the function returns, returns sensible data..

Reply:
Fixed